Backend/Plugins/IRacingPlugin.cs

```diff
@@ -85,6 +85,8 @@ private class CarState
             { "DirtOval", IRacingCategoryEnum.DirtOval },
         };
 
+        private readonly int[] LastPositionInClass = new int[64];
+        private readonly int[] LastPositionInRace = new int[64];
```

Comment:
Should we introduce constants, I suspect 64 is MAXCARONTRACK constant of some sort. not super important at the moment, just a thought.

Reply:
You are absolutely right. I dont like magic numbers, so added a MAX_CARS constant and used that instead.